lib/BUILD

```diff
@@ -199,7 +199,10 @@ cc_library(
     name = "jpegxl",
     srcs = libjxl_dec_sources + libjxl_dec_box_sources + libjxl_dec_jpeg_sources + libjxl_enc_sources + libjxl_cms_sources,
     compatible_with = DEFAULT_COMPATIBILITY,
-    defines = ["JPEGXL_ENABLE_SKCMS=1"],
+    defines = [
+        "JPEGXL_ENABLE_SKCMS=1",
+        "FJXL_STANDALONE=0",
```

Comment:
Doesn't that get defined automatically? At least there is some logic for it somewhere (possibly the .cc file?)

Reply:
Yes, there is, but apparently that does not work in bazel, and this seemed to be the easiest way to fix.

Reply:
That seems mildly suspicious, is perhaps some other define missing?

Reply:
Defining the version macros works too.

Reply:
Yeah, that seems better - although perhaps it fits best in a separate PR...